docs/guide/features.md

```diff
@@ -124,6 +119,8 @@ Vite's default types are for its Node.js API. To shim the environment of client
 /// <reference types="vite/client" />
 ```
 
+::: details Using `compilerOptions.types`
```

Comment:
Since there's a caveat, I think we can put this inside a details block so users can skip reading it in most cases.